### ISSUE TITLE
Minimal API Scaffold generates a wrong Created location

### ISSUE THREAD

**Issue by brunolins16** [dotnet/Scaffolding#1963]
The scaffolder generates the following `POST` endpoint:

``` c#
        routes.MapPost("/api/SampleModel/", async (SampleModel sampleModel, SampleModelContext db) =>
        {
            db.Samples.Add(sampleModel);
            await db.SaveChangesAsync();
            return Results.Created($"/SampleModels/{sampleModel.Id}", sampleModel);
        })
```

However, the `location` provided to the `Results.Created` seems to be wrong, since it does not match with the `GET` endpoint generated:

``` c#
routes.MapGet("/api/SampleModel/{id}", async (int Id, SampleModelContext db) =>
        {
            return await db.Samples.FindAsync(Id)
                is SampleModel model
                    ? Results.Ok(model)
                    : Results.NotFound();
        })
```

Should the `location` be `$"api/SampleModel/{sampleModel.Id}"` or at least  `$"SampleModel/{sampleModel.Id}"`?

**Comment by deepchoudhery** [comment]
Yup, fixed in [this ](https://github.com/dotnet/Scaffolding/pull/1986)PR for 7.0-RC1, servicing fix for 6.0 soon.

**Comment by vijayrkn** [comment]
@deepchoudhery - Please validate and close.

**Comment by deepchoudhery** [comment]
fixed in 7.0-rc1 and 6.0.10. Closing.
